packages/react/src/components/Popup/Popup.tsx

```diff
@@ -227,7 +231,7 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
         {this.state.open &&
           Popup.isBrowserContext &&
           popupContent &&
-          ReactDOM.createPortal(popupContent, document.body)}
+          (inline ? popupContent : ReactDOM.createPortal(popupContent, document.body))}
```

Comment:
Can we move this change to a separate PR? And cover it with UT

Reply:
Sure thing, I've even added new FocusZone component.. All of these component related changes will be moved to separate PRs, once the accessibility team confirms that this work as expected.